package.json

```diff
@@ -6,23 +6,17 @@
     "node": "16"
   },
   "scripts": {
-    "build": "react-app-rewired build",
     "dev": "run-p dev:*",
```

Comment:
I guess `dev` might not be needed anymore. We already have `dev:server": "nodemon --exec \"node server/index.js`. Or maybe just rename `dev:server` to `dev`